search.php

```diff
@@ -14,6 +14,9 @@
 		<section class="travel-results pb1 md-pt1">
 			<div class="travel-inline-list travel-results-list">
 				<div class="max-width-3 mx-auto px1 md-px2">
+					<?php if ( isset( $_GET['s'] ) ) : ?>
+						<h1><?php printf( '%d Results for <strong>%s</strong>', esc_html( amp_travel_get_posts_count() ), esc_html( $_GET['s'] ) ); ?></h1>
```

Comment:
The search template is looking good. How about using `_n()` to differentiate between singular and plural here?